src/components/Icons/Elipse.tsx

```diff
@@ -0,0 +1,28 @@
+import { type IconProps } from './types'
+import { DEFAULT_ICON_COLOR, DEFAULT_ICON_SIZE } from './constants'
+
+export function ElipseIcon({
+  'aria-label': ariaLabel = 'Elipse Icon',
+  size = 28,
```

Comment:
```suggestion
  size = DEFAULT_ICON_SIZE,
```
I would use the DEFAULT_ICON_SIZE